readthedocs.yml

```diff
@@ -1,5 +1,8 @@
-conda:
-  file: .rtd-environment.yml
-
 python:
-   setup_py_install: true
+  version: 3.5
```

Comment:
```suggestion
  version: 3.7
```
Python 3.7 is faster, if available.

Reply:
Actually, nvm. I vaguely remember some Sphinx stuff not working in 3.7. Try 3.6.